CartoDB/cartodb
# Fixing #11308 problem

Ticket: #11308.

The problem is great described about: "Previous style is applied after reopening a map and applying a new style". We were resetting the styles whenever the layer was saved, and it was not correct.

**Things to test:**
- Check the steps mentioned in the ticket above.
- Check if after applying an autostyle, if user makes any cartocss change, autostyle is reseted correctly.
